src/components/paragraph-modifiers.tokens.json

```diff
@@ -0,0 +1,26 @@
+{
+  "oip": {
+    "utrecht": {
+      "paragraph": {
+        "small": {
+          "font-size": {
+            "value": "14px"
```

Comment:
As I noted in your related PR, I'd prefer variable names with the size indicator at the end: `oip-utrecht-paragraph-font-size-small`, `oip-utrecht-paragraph-font-size-medium`, and so on. Differing strings are easier to discern if the differing part is at the end.

Reply:
As noted in [OIP PR nr.1215](https://github.com/maykinmedia/open-inwoner/pull/1215): NL Design System works differently. 
It first defines the modifier from a component (in this case 'small') and then it sets the attributes for those modifiers (in this case 'line-height'), that then finally will receive their value from JSON.
I know it is confusing, but these are SCSS variables, which are not the same as HTML classnames.

Reply:
Oh yeah, and another season why you don't want to have dash-separated stuff that all start with the same 'name' but need different values, is because the **style-dictionary** package would merge and throw away everything that starts with `font-size`-blah-blah, so all that start with `font-size-small`/`font-size-default`/`font-size-large`.. .etc. would just be ignored and not be part of the build.
So that is why in the Readme it says the structure needs to be `<prefix>.<component>.<modifier>.<UIState>.<CSSProperty>`

Reply:
That's a good explanation. Makes sense now.